charts/rasa-x/templates/event-service-deployment.yaml

```diff
@@ -29,7 +29,11 @@ spec:
       {{- end }}
       containers:
       - name: {{ .Chart.Name }}
+        {{- if not .Values.eventService.tag }}
+        image: "{{ .Values.eventService.name }}:{{ .Values.rasax.tag | default .Chart.AppVersion }}"
```

Comment:
how about using a helper template for `{{ .Values.rasax.tag | default .Chart.AppVersion }}` to not repeat ourselves?
Doing this we could also unify lines 33 and 35, e.g. something along the lines:

```
        image: "{{ .Values.eventService.name }}:{{ .Values.eventService.tag | default <template function> }}"

```

Reply:
Sounds good! I see one place more where we can replace the `rasax.tag` variable and use a template function instead.